dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd

```diff
@@ -63,6 +64,7 @@ Export-Package: \
   !org.apache.myfaces.shared.renderkit, \
   !org.apache.myfaces.shared.renderkit.html, \
   !org.apache.myfaces.shared.renderkit.html.util, \
+  org.apache.commons.digester, \
```

Comment:
I'm not sure if it's necessary to export the package? Other bundles don't and these digester packages are treated as private.

Reply:
No issues when I removed it. Everything resolved in my local build.